### PR TITLE
[Bn Translation] Location Tracking

### PR DESCRIPTION
**Pull request by Iftu119**
With this PR, 

* From `3-transport` , 

  * The topmost [Readme](https://github.com/microsoft/IoT-For-Beginners/blob/main/3-transport/README.md) for `transport` has been translated and the placeholder .dummy has been removed.
  * The [Readme](https://github.com/microsoft/IoT-For-Beginners/blob/main/3-transport/lessons/1-location-tracking/README.md) for `Location Tracking` and corresponding [assignment](https://github.com/microsoft/IoT-For-Beginners/blob/main/3-transport/lessons/1-location-tracking/assignment.md) have been translated  along with removing dummy.
 
* The [Bengali translation of main Readme](https://github.com/microsoft/IoT-For-Beginners/blob/main/translations/README.bn.md) has been updated with the links.

### CODE REVIEW

File: 3-transport/translations/README.bn.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+> 🎓 সাপ্লাই চেইনের পরিবহণের অংশকে বলা হয় *logistics* ।
+
+এই চারটি পাঠে, আমরা দেখবো কীভাবে খাদ্য (ভার্চুয়াল) ট্রাকে বোঝাই করে খাবারের তদারকির মাধ্যমে সরবরাহের চেইনের উন্নতি করতে ইন্টারনেট অফ থিংস প্রয়োগ করা যায়, যেখানে যানবাহনের গন্তব্যস্থলে পৌঁছানো  পর্যন্ত ট্র্যাক করা হয়। আমরা জিপিএস ট্র্যাকিং, জিপিএস ডেটা কীভাবে সংরক্ষণ এবং ভিজ্যুয়ালাইজ করব এবং কীভাবে কোন ট্রাক তার গন্তব্যস্থলে পৌঁছালে ঠিকমতো তার ব্যাপারে জানানো যায় তা শিখবো।
+
+> 💁  এই লেসনগুলোতে আমরা কিছু ক্লাউড রিসোর্স ব্যবহার করবো। এখানে সবগুলো প্রজেক্ট সম্পূর্ণ না হলেও, আমাদেরকে [প্রজেক্ট ক্লীন-আপ](../translations/clean-up.bn.md) করতে হবে।
+
+## পাঠসমূহ
+
+1. [লোকেশন ট্র্যাকিং](lessons/1-location-tracking/translations/README.bn.md)
```
Comment (jimbobbennett) [inline 3-transport/translations/README.bn.md:17]: Are these links correct? Shouldn't they be up one level?

File: 3-transport/lessons/1-location-tracking/translations/assignment.bn.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# অন্যান্য জিপিএস ডেটা পর্যালোচনা করা
+
+## নির্দেশাবলী
+
+আমাদের জিপিএস সেন্সর থেকে আসা NEMA বাক্যগুলিতে লোকেশন ছাড়াও অন্যান্য ডেটা রয়েছে। অতিরিক্ত ডেটা অনুসন্ধান করে  আইওটি ডিভাইসে তা ব্যবহার করি।
+
+উদাহরণস্বরূপ - আমরা কী বর্তমান তারিখ এবং সময় পেতে পারি? আমরা যদি কোন মাইক্রোকন্ট্রোলার ব্যবহার করি, তবে  যেভাবে পূর্বের প্রজেক্টে এনটিপি সিগন্যাল ব্যবহার করেছি সেটাকে ব্যবহার করে জিপিএস ডেটা দ্বারা ঘড়ি সেট করা যাবে? আমরা কি উচ্চতা (সমুদ্রতল থেকে আমাদের উচ্চতা) বা বর্তমান গতি পেতে পারি?
+
+If you are using a virtual IoT device, then you can get some of this data by sending NMEA sentences generated using tools [nmeagen.org](https://www.nmeagen.org).
```
Comment (jimbobbennett) [inline 3-transport/lessons/1-location-tracking/translations/assignment.bn.md:9]: Does this need translating?

**Review by jimbobbennett — CHANGES_REQUESTED** [review]

File: 3-transport/translations/README.bn.md (hunk 1 above)
Comment (Iftu119) [reply 3-transport/translations/README.bn.md]: Thanks. They have been raised one level up.

File: 3-transport/lessons/1-location-tracking/translations/assignment.bn.md (hunk 2 above)
Comment (Iftu119) [reply 3-transport/lessons/1-location-tracking/translations/assignment.bn.md]: I translated this in the next line but forgot to remove the English sentence.   Thanks a lot for this catch. ❤️

**Review by jimbobbennett — APPROVED** [review]

**Pull request merged**